gem/lib/rbi-central/static/context.rb

```diff
@@ -19,8 +19,12 @@ def initialize(gem, annotations_file, color:, bundle_config: {})
       def init!
         super
 
-        add_gem_dependency("sorbet", version: ">= 0.5.10109")
-        add_gem_dependency("tapioca", version: ">= 0.9.2")
```

Comment:
This was causing an issue, because `">= 0.9.2"` wouldn't select for a prelease version like `0.17.2`.

@Morriar is there any downside to using the same gem versions as the `rbi-central` gem itself uses?